### PR TITLE
Adding new VirtualMachines.POST.Reapply action to compute api-version: 2019-07-01. Updating parameter info for skus api-version:2019-04-01.. 

### PR DESCRIPTION
**Pull request by avinashakka**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -2832,6 +2832,46 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Please add an example file.

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Added.

**Comment by yungezz** [comment]
hi @avinashakka since this is to merge into a private branch, what level of reviewing you'd like to have? an official reviewing as a normal spec PR or your internal team reviewing?

**Comment by avinashakka** [comment]
> hi @avinashakka since this is to merge into a private branch, what level of reviewing you'd like to have? an official reviewing as a normal spec PR or your internal team reviewing?

Hi @yungezz, Official. I have reviewed this change with my team and @hyonholee. Please go ahead and review it.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
@avinashakka 
PR look good to me. If you do not need ARM review, please just resolve the conflict on base branch and we could get it merged.

**Pull request merged**